js/service/shell/index.js

```diff
@@ -44,3 +44,20 @@ exports.setCommand = function(name, cb) {
   assert(typeutils.isFunction(cb));
   commands.set(name, cb);
 };
+
+exports.runCommand = function(name, args, cb) {
+  var capture = require('./captureio.js');
```

Comment:
Well, no. It's intended to be used by the user therefore (I think) it should accessible from `runtime.shell`. I may have understood you wrong, so please clarify.